docs/admin/README.md

```diff
@@ -86,12 +86,29 @@ With the 0.1.0 release, the Teams page just lists the teams on the platform.
 
 Further team management options will come in later releases.
 
+### Managing Project Types
+
+The Project Types page can be used to manage the Project Types on the platform.
+
+When billing is enabled, a project type can be associated with a particular
+Stripe Product/Price - allowing each type to have a different monthly price
+associated with it.
+
+The Project Types page shows what types are current active, how many stacks
+each type has assigned to it, and how many projects have been created of that
+type.
+
 ### Managing Stacks
 
 The Stacks page can be used to manage the Project Stacks on the platform.
 
 It can be used to create and edit the stacks on the platform.
 
+With the 0.7 release, you can create a new version of an existing stack via the
```

Comment:
Should we mention version numbers in docs, feels like its just a way to go out of date, we have the release notes and blog post that shows when something was released.